homeassistant/components/recorder/util.py

```diff
@@ -172,7 +172,8 @@ def execute(
                 raise
             time.sleep(QUERY_RETRY_WAIT)
 
-    assert False  # unreachable # pragma: no cover
+    # Unreachable
+    raise RuntimeError  # pragma: no cover
```

Comment:
Can we remove `pragma no cover` here?
```suggestion
    raise AssertionError("Should be unreachable")
```

Reply:
```suggestion
    raise RuntimeError("Should be unreachable")
```